File: Exercicios-Beecrowd/Kotlin/1015.kt
import java.util.*
import kotlin.math.*

fun main(args: Array<String>) {

    val a = readLine().toString().split(" ")
    val b= readLine().toString().split(" ")
    val x1=a[0].toDouble()
    val x2=a[1].toDouble()
    val x3=b[0].toDouble()
    val x4=b[1].toDouble()
    val resultado=(((x3-x1).pow(2))+((x4-x2).pow(2))).pow(0.5)
    println("%.4f".format(resultado))
	
}

File: Exercicios-Beecrowd/C#/1015.cs
using System; 

class URI {

    static void Main(string[] args) { 
            string[] lista1 = Console.ReadLine().Split(' ');
            string[] lista2 = Console.ReadLine().Split(' ');
            double a = Convert.ToDouble(lista1[0]);
            double b = Convert.ToDouble(lista1[1]);
            double c = Convert.ToDouble(lista2[0]);
            double d = Convert.ToDouble(lista2[1]);
            double valor = Math.Pow((Math.Pow((c - a), 2) + Math.Pow((d - b), 2)), 0.5);
            string resposta = valor.ToString("F4");
            Console.WriteLine(resposta);
    }

}